wiki/start/simple-game-extended.md

```diff
@@ -21,18 +21,29 @@ We will start by creating a `Drop` class, which extends Game and whose `create()
 package com.badlogic.drop;
 
 import com.badlogic.gdx.Game;
+import com.badlogic.gdx.Gdx;
 import com.badlogic.gdx.graphics.g2d.BitmapFont;
 import com.badlogic.gdx.graphics.g2d.SpriteBatch;
+import com.badlogic.gdx.utils.viewport.FitViewport;
 
 
 public class Drop extends Game {
 
 	public SpriteBatch batch;
 	public BitmapFont font;
+	public FitViewport viewport;
+
 
 	public void create() {
 		batch = new SpriteBatch();
-		font = new BitmapFont(); // use libGDX's default Arial font
+		// use libGDX's default Arial font
```

Comment:
It's now Liberation Sans. "Default font" would be fine, as it isn't really relevant to the tutorial what it is. My eyes aren't working - you'd have to get @raeleus to see about the rest.